src/pages/Confirm/Confirm.styled.jsx

```diff
@@ -29,41 +29,47 @@ export const RatingImg = styled.img`
 `
 export const CheckImg = styled.img`
   width: auto;
-  margin: 0 30px;
+  margin-top: 10px;
+  // margin: 0 30px;
 
   @media (max-width: 960px) {
     position: relative;
     height: 100px;
-    bottom: -20px;
-    width: 94px;
+    // bottom: -20px;
+    width: 80px;
+    height: auto;
   }
```

Comment:
Please avoid to mix `max width` with `min width` media queries.

You can try to use:

```css
  width: 80px;
  margin-top: 10px;
  height: auto;
  position: relative;
```

instead of use:

```css
 width: auto;
  margin-top: 10px;
  // margin: 0 30px;
  @media (max-width: 960px) {
    position: relative;
    height: 100px;
    // bottom: -20px;
    width: 80px;
    height: auto;
  }
```

Here we've:

1. Removed unused code (width auto, height 100px and comments code)
2. Removed unnecessary media max-width block code